docs/Database_Schema.md

```diff
@@ -171,3 +171,24 @@ CREATE TABLE "validators" (
     PRIMARY KEY("enrolled_at","utxo_key")
 )
 ```
+
+----
+
+## 6. Table **information**
+
+### _Schema_
+
+| Column            | Data Type | PK | Not NULL | Default  |Description|
+|:----------------- |:--------- |:--:|:--------:| -------- | --------- |
+|  key              | TEXT      | Y  | Y        |          | The key   |
+|  value            | TEXT      |    | Y        |          | The value |
+
+### _Create Script_
+
+```sql
+CREATE TABLE informations (
```

Comment:
The Table name alone is difficult to estimate the purpose of this table.
It would be better to add a more specific table name or description.

Reply:
I changed it